### DISCUSSION TITLE
PHP 8.1 support

### DISCUSSION THREAD

**Discussion by brendt** [spatie/laravel-enum#85] [General]
We need to discuss what's needed in order to support PHP 8.1. The base enum package will change significantly, and Laravel has built-in support for native PHP enum casts. So we'll probably need a major rewrite as well of this package in order to support the new enum package and native PHP enums.

**Comment by brendt** [comment]
@Gummibeer I'd appreciate your input.

**Comment by Gummibeer** [comment]
I've seen the native enum cast. Right now it doesn't support to cast an array of enums.
First I would finish the base num package for PHP8.1 and release it and after that I would check what has to be changed in this package.

**Reply by brendt** [reply]
I agree!
